src/relation/mod.rs

```diff
@@ -963,7 +965,71 @@ impl Variant for Set {
         vec![&self.left, &self.right]
     }
 }
+/// Literal
+#[derive(Clone, Debug, Hash, PartialEq, Eq)]
+pub struct Literal {
+    /// The name of the output
+    pub name: String,
+    /// The value
+    pub value: Value,
+    /// The schema description of the output
+    pub schema: Schema,
+    /// The size of the Set
+    pub size: Integer,
+}
+
+impl Literal {
+    pub fn new(name: String, value: Value) -> Self {
```

Comment:
No we do not want a `Literal`, `Expr::Value` is already playing this role.
We want a Relation holding a `Vec` of `Value`s of the same `DataType` (given by the schema, be it DataType::Any if needed).
Let's call it `Values` as discussed initially.

Size is obvious from the len of the Vec. No need to use `List` and give artificially a size to all `Value`s.
A `Relation` is by definition a bunch of values with the same datatype given by the schema.